include/openssl/bio.h.in

```diff
@@ -76,7 +76,7 @@ extern "C" {
  * BIO_set_fp(in,stdin,BIO_NOCLOSE);
  */
 # define BIO_NOCLOSE             0x00
-# define BIO_CLOSE               0x01
+# define BIO_CLOSE               BIO_FLAGS_CLOSE
```

Comment:
This is changing public API/ABI. Please don't.

Reply:
I meanwhile found a better solution...